x-pack/plugins/actions/server/lib/action_executor.ts

```diff
@@ -313,6 +314,17 @@ export class ActionExecutor {
         }
         // end gen_ai extension
 
+        // start sentinelone extension
```

Comment:
In order to be able to display results for actions related to the exact alert we need to be able to store `params` which contains information about the `subAction` type, as well as `data` which is the response data coming from the SentinelOne API, which for `executeScript` `subAction` returns `parentTaskId`, which we have to use in order to receive the status of the async `executeScript` action